plugins/deye_plugin_ha_discovery.py

```diff
@@ -130,12 +130,12 @@ def _get_device_class(topic: str) -> str:
 
         # topic: ac/l*/voltage
         # topic: dc/pv*/voltage
-        if topic.endswith("/voltage"):
+        if topic.endswith("/voltage") or topic.endswith("/charging_voltage") or topic.endswith("/discharge_voltage"):
```

Comment:
Please expand the topics everywhere in the comment above the if statement to simplify later changes.

Reply:
Done in ce0026a